config/main.go

```diff
@@ -49,6 +50,7 @@ var Keys = EntitlementsConfigKeysType{
 	Port:            "PORT",
 	CertsFromEnv:    "CERTS_FROM_ENV",
 	SubsHost:        "SUBS_HOST",
+	ComplianceHost:  "COMPLIANCE_HOST",
```

Comment:
We should probably also add this in our template, that way we can define it per environment in app-interface? - https://github.com/RedHatInsights/entitlements-api-go/blob/master/deployment/deploy.yml

Reply:
- [ ] add to app interface template